tests/test_main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from app.main import cryptocurrency_action
+from unittest import mock
+import pytest
+
+
+@pytest.mark.parametrize(
+    "exchange_rate,current_rate,expected_result",
+    [
+        pytest.param(
+            1.0, 1.1,
+            "Sell all your cryptocurrency",
+            id="test prediction for decreasing"
+        ),
+        pytest.param(
+            1.1, 1.0,
+            "Buy more cryptocurrency",
+            id="test prediction for increasing"
+        ),
+        pytest.param(
+            1.30, 1.26,
+            "Do nothing",
+            id="test without changes -5%"
+        ),
+        pytest.param(
+            1.8, 1.84,
+            "Do nothing",
+            id="test without changes +5%)"
```

Comment:
Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

Reply:
You have not fixed my comment. 1.8 / 1.85 is 0.97297... not **exactly** 0.95.